bin/hledger-prices.hs

```diff
@@ -19,9 +20,10 @@ import Hledger.Cli
 ------------------------------------------------------------------------------
 cmdmode = hledgerCommandMode
   [here| prices
-Print all prices from the journal.
+Print all market prices from the journal.
   |]
-  [flagNone ["costs"] (setboolopt "costs") "print transaction prices from postings instead of market prices"]
+  [flagNone ["costs"] (setboolopt "costs") "print transaction prices from postings"
+  ,flagNone ["inverted-costs"] (setboolopt "inverted-costs") "print transaction inverted prices from postings also"]
   [generalflagsgroup1]
```

Comment:
Useful discussion, highlighting that, although the essential maths are the same, how you write the journal entry - eg the order of postings, or which posting you write the price on - can affect report output. 

But, sorry, I don't follow what you mean with the second part ("consider that I'm abroad...") ?

What's your opinion on consistently using "price" and "transaction price" (or something better) rather than "cost" ?

As the world's most knowledgeable user of this command, may I ask you: in your real world usage, when/why do you use this command, the --cost flag, and the --inverted-cost flag ?